src/lwc/geFormField/__tests__/geFormField.test.js

```diff
@@ -204,28 +204,6 @@ describe('c-ge-form-field', () => {
 
 });
 
-expect.extend({
```

Comment:
Migrated this to jest.setup.js, so we can use it in all tests. @psadie-sfdo

Reply:
Sounds good! Thanks for the heads up!